### DISCUSSION TITLE
BloatyNosyApp 0.51

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#40] [Announcements]
- Numerous UI improvements
- App display is centered, similar to the Windows 11 Start menu
- The old "BloatFinder" module has been extensively revamped at its core and is now called "AppyTrash"
- Numerous minor bug fixes

Thank you very much for the lively interest in the store version and the numerous ratings and feedback.

**Please note that the version published on GitHub is more up-to-date than the one in the Microsoft store. Updates in the store must always be approved by Microsoft, which can take up to three days.**

[<img alt="alt_text" width="240px" src="https://upload.wikimedia.org/wikipedia/commons/f/f7/Get_it_from_Microsoft_Badge.svg" href=""/>](https://apps.microsoft.com/store/detail/XPDCJG88N6FD16) 
_Microsoft Store deep link: ms-windows-store://pdp/?productid=XPDCJG88N6FD16_

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/BloatyNosy/releases/tag/0.51.0'>BloatyNosyApp 0.51</a>.</em>

**Comment by Toofast520** [comment]
Sounds great, one question, with all the issues with MSI here recently is there any chance the your program was compromised. Also, because I’m stupid, what’s that particular download for? Last but not least, is this free or is there a pay option?  Thanks for your time!

**Reply by tpkowastaken** [reply]
It's on github therefore it's free.

**Reply by Toofast520** [reply]
Believe it or not I wanted to pay the guy. All the work for what? A free service that’s helpful for no charge…. Bummer for him. Thanks for the reply!

**Reply by tpkowastaken** [reply]
Yeah. He can setup sponsor button that would sit next to the watch button up there. But my friend looked into it and it apparently doesn't do much. And for some reason there is a "disable throttling" which does the exact opposite of what you should be doing. Throttling restricts applications like your browser from using too much power when you're gaming for instance. If you want to optimize for gaming you could try [a program](https://github.com/tpkowastaken/performance-mode-for-all/releases/) that sets all apps except for the game to efficiency mode. This brutally improved performance in windows 11. In csgo it improved my performance from fps drops to 5fps average 70 to fps drops to 110 with average 170fps. Also the input lag was gone. You should give it a try.
